common/rest-jdbc-spring-api/build.gradle

```diff
@@ -7,6 +7,7 @@ apply from: project(":").file('gradle/java.gradle')
 dependencies {
     implementation group: 'io.swagger', name: 'swagger-annotations'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
```

Comment:
qq - what is this for please?

Reply:
Spring Boot 2.3.0 web starter doesn't include the `javax.validation.*` package. Hence we need to include the `spring-boot-starter-validation` as a dependency https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.3-Release-Notes#validation-starter-no-longer-included-in-web-starters

Reply:
See commit b590e732bdb4544f56ba4b63632a28ef792cf94c for more info